### ISSUE TITLE
Form options not sending with POST request

### ISSUE THREAD

**Issue by fcosrno** [germanrio/request-cached#5]
It looks like the form option is not working for POST methods. Is this accurate?  The following wouldn't work. It's not sending the form params.

``` javascript
requestCached({
    main: {
        uri: 'http://example.com',
        method: 'post',
        form:{name:'something'}
    }
}
```

**Comment by germanrio** [comment]
I will take a look into it, when I find a bit of time.

**Comment by germanrio** [comment]
Fixed in version 1.0.0
